### ISSUE TITLE
dataset attributes

### ISSUE THREAD

**Issue by Favourte** [caoji2001/HOSER#2]
Thank you for your brilliant work. Could you explain the purpose of each file in each dataset in the data folder and the attributes in the file?

![Image](https://github.com/user-attachments/assets/5b381a41-dea4-4cdc-aac4-6cff2f896ed3)

**Comment by caoji2001** [comment]
Thank you for your interest in our work. The meanings of each file are as follows:

`road_network_partition`: The result of partitioning the road network into zones using KaHIP.
`roadmap.geo`: Information about each road segment in the network.
`roadmap.rel`: The topological structure of the road network (how road segments are connected).
`train.csv`, `val.csv`, and `test.csv`: Used for the trajectory data of the training set, validation set, and test set.
`zone_trans_mat.npy`: Traffic flow information between zones.
